packages/zone.js/lib/zone.configurations.api.ts

```diff
@@ -0,0 +1,669 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+/**
+ * Interface of zone.js configurations.
+ *
+ * You can define the following configurations on the window/global object before
+ * importing `dist/zone.js` to change zone.js default behaviors.
+ */
+interface ZoneGlobalConfigurations {
+  /**
+   * Disable the monkey patch of the `Node.js` `EventEmitter` API.
+   *
+   * By default, the `zone.js` monkey patches the `Node.js` `EventEmitter` APIs to make async
+   * callbacks of those APIs in the same zone when scheduled.
+   *
+   * Consider the following example:
+   *
+   * const EventEmitter = require('events');
+   * class MyEmitter extends EventEmitter {}
+   * const myEmitter = new MyEmitter();
+   *
+   * const zone = Zone.current.fork({name: 'zone'});
+   * zone.run(() => {
+   *   myEmitter.on('event', () => {
+   *     console.log('an event occurred!', Zone.current.name);
+   *     // the callback runs in the zone when it is scheduled,
+   *     // so the output is 'an event occurred zone'.
+   *   });
+   * });
+   * myEmitter.emit('event');
+   *
+   * If you set `__Zone_disable_EventEmtter = true` before importing zone.js,
+   * `zone.js` does not monkey patch the `EventEmitter` API and the above code
+   * outputs 'an event occurred <root>'.
```

Comment:
Nit: make console more readable
```suggestion
   * outputs `an event occurred in zone: <root>`.
```

Please change other examples to make them consistent.

Reply:
Got it, updated, thanks.